specifyweb/frontend/js_src/lib/components/Interactions/InteractionDialog.tsx

```diff
@@ -329,6 +339,7 @@ export function InteractionDialog({
                   )}
                   <div>
                     <Button.Blue
+                      disabled={disableIgnore}
```

Comment:
Can you please clarify the behavior you are going for (under which conditions the button should be enabled or disabled). It would also be useful for testers.
Feel free to double-check with Grant for what's expected.
Looking at the code, it looks like `setDisableIgnore(true);` is called if there are any missing items. It is not called if no preparations where found.